distribution/ddf-common/src/main/resources/etc/fipsToIso.properties

```diff
@@ -0,0 +1,249 @@
+# FIPS 10-4 to ISO 3166-1 alpha 3 country codes from http://www.opengeocode.org/
```

Comment:
Instead of putting this file here, we should keep it inside the bundle and copy it when it gets installed (see https://github.com/codice/ddf/blob/master/platform/security/security-services-app/src/main/resources/features.xml#L96 for an example).